CMake/FindCubeb.cmake

```diff
@@ -0,0 +1,14 @@
+find_path(CUBEB_INCLUDE_DIR cubeb.h PATH_SUFFIXES cubeb)
+find_library(CUBEB_LIBRARY cubeb)
+mark_as_advanced(CUBEB_INCLUDE_DIR CUBEB_LIBRARY)
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(CUBEB DEFAULT_MSG
+	CUBEB_INCLUDE_DIR CUBEB_LIBRARY)
+
+if(CUBEB_FOUND AND NOT TARGET CUBEB)
+  add_library(cubeb::cubeb SHARED IMPORTED)
+  set_target_properties(cubeb::cubeb PROPERTIES
```

Comment:
@Orphis, shouldn't this also define an `IMPORTED_LOCATION` as well?

Reply:
Also, should this be `SHARED` or `UNKNOWN`? I notice most of the other examples in dolphin's repo are using `UNKNOWN`.

Reply:
Correct! I totally missed that and assumed it was tested.
Technically, this isn't searching for a shared library but a system one. It could find a static version too, so it should be UNKNOWN. Maybe the output message should be changed too to reflect that?

Reply:
Thanks for the review! I suppose the output message should be changed for other libraries too, but that is out of the scope of this PR. I'll make the other changes, test locally and then update this PR in a little bit.